README.md

```diff
@@ -212,6 +212,19 @@ You will likely see a number of create table statements (see data model section
 which tells you that if you apply these changes, that sql script will be applied to the sample db
 
 
+### Database servers with passwords
+
+  sem-apply --url postgresql://postgres@localhost/sample --pgpass
```

Comment:
is the intent to pass in "--pgpass <password>" ? or just a flag that then prompts for the password?